BlackJackTests/GameTests.cs

```diff
@@ -210,99 +214,136 @@ public void TestIfCalculateScoreReturnsTotalSumOfPlayersHandWithFiveCards(string
+        /*[Fact]
+        public void TestIfWinOrLossDuringGamePrintsCorrectlyWhenItsDealersTurnAndScoreIsOver21()
+        {
+            using (StringWriter sw = new StringWriter())
             {
-                "5 of Diamonds",
-                "10 of Clubs",
-                "3 of Clubs",
-                "4 of Clubs",
-                "5 of Clubs",
-                "6 of Clubs",
-                "7 of Clubs",
-                "8 of Clubs",
-                "9 of Clubs",
-                "Kind of Diamonds",
-                "Jack of Clubs",
-                "Queen of Clubs",
-                "Kind of Clubs",
-                "Ace of Clubs",
-                "2 of Diamonds",
-                "3 of Diamonds",
-                "4 of Diamonds",
-                "6 of Diamonds",
-                "7 of Diamonds",
-                "8 of Diamonds",
-                "9 of Diamonds",
-                "10 of Diamonds",
-                "Jack of Diamonds",
-                "Queen of Diamonds",
-                "8 of Hearts",
-                "Ace of Diamonds",
-                "2 of Hearts",
-                "3 of Hearts",
-                "4 of Hearts",
-                "5 of Hearts",
-                "6 of Hearts",
-                "7 of Hearts",
-                "5 of Spades",
-                "9 of Hearts",
-                "10 of Hearts",
-                "Jack of Hearts",
-                "2 of Clubs",
-                "Queen of Hearts",
-                "Kind of Hearts",
-                "Ace of Hearts",
-                "2 of Spades",
-                "3 of Spades",
-                "4 of Spades",
-                "Jack of Spades",
-                "6 of Spades",
-                "7 of Spades",
-                "8 of Spades",
-                "9 of Spades",
-                "10 of Spades",
-                "Queen of Spades",
-                "Kind of Spades",
-                "Ace of Spades"
-            };
+                Console.SetOut(sw);
+                
+                Game game = new Game(new ConsoleActions(), new Deck(new ConsoleActions()));
+                Deck deck = new Deck(new ConsoleActions());
+                int score = 30;
+                var gamersHand = new List<Card>();
+                gamersHand.AddRange(new[]
+                {
+                    new Card() {Suite = "Diamonds", Value = "8"},
+                    new Card() {Suite = "Hearts", Value = "Ace"}
+                });
+                
+                game.WinOrLossDuringGame(score, gamersHand);
 
-            List<string> dealersHand = new List<string>() {"8 of Spades", "4 of Diamonds"};
-            
-            var deckMock = new Mock<IDeck>();
-            deckMock.Setup((s => s.DrawCard()))
-                .Returns("Queen of Diamonds");
+                string expected = string.Format("Player Wins!{0}", Environment.NewLine);
+                Assert.Equal(expected, sw.ToString());
+            }
+        }*/
+        
+        
+        [Theory]
```

Comment:
what's the difference between [Theory] and [Fact]?

Reply:
Theory is to do multiple test cases for the same test method but with Fact you can only do one.